src/Reshuffle.ts

```diff
@@ -157,4 +157,12 @@ export default class Reshuffle {
   getPersistentStore() {
     return this.registry.common.persistentStore
   }
+
+  public clearInterval(intervalID: NodeJS.Timer): void {
```

Comment:
Can't those functions stay in the CronService?

Reply:
This is used by multiple services that need intervals, for example S3 if polling the state of different buckets to track file additions and removals. I wanted services to refrain from using the built in methods, so that in the future we can replace the internal time with an external signal, that can be load balanced across multiple runtime instances